sdk/lusid/api_client.py

```diff
@@ -295,6 +300,9 @@ def __deserialize(self, data, klass):
         """
         if data is None:
             return None
+        
+        if self.flag_to_use_decimal:
+            self.NATIVE_TYPES_MAPPING['float'] = Decimal
```

Comment:
If we have this then the call of `if type(klass) is float....` part in `self.__deserialize_primitive` will never be called as all `floats` will resolve to `Decimal` which means `if type(klass) is float` will always be false.

Reply:
That's a good point, I have changed the condition to if type(klass) = Decimal